www/container/list/index.js

```diff
@@ -231,11 +231,21 @@ var updateAddressBar = function() {
     var clickedPage = s('#pageNumber').val();
     // Current page
     var currentPage = (window.location.pathname).split('/');
+    // Current structure id
+    var currentStructure = s('#navigationId').val();
+
+    // Default url (if standard output)
+    var url = currentPage[1] + '/' + currentPage[2] + '/' + clickedPage;
```

Comment:
@getmanenko You are still parsing current url instead of using parameters that we have passed as inputs